server/activityItem/fetch.ts

```diff
@@ -135,7 +135,7 @@ const fetchActivityItemModels = async (
 
 const getActivityItemAssociationIds = (
 	items: types.ActivityItem[],
-	scope: Scope,
+	scope?: Scope,
```

Comment:
What does an optional scope resolve?

Reply:
The `Scope` here provides some IDs for a Community/Collection/Pub trio — if we pass them in, those models will be included in the result, otherwise they won't. We pass a `Scope` in for the Activity dashboard (which belongs to a scope) but not for the notifications tray (which doesn't).